hack/generate-crd-version-patches.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
Script for breaking up CRD files into base and patch files, where we have patch files for Open API schema for every API version.